src/referenced-fields.ts

```diff
@@ -403,6 +404,7 @@ const countTableSelects = (
 		case 'AddDateNumber':
 		case 'DateTrunc':
 		case 'Equals':
+		case 'ExtractJSONPathAsText':
```

Comment:
Added ExtraJsonPathAsText as a Binary node - The first argument will always be a JSON and therefore return zero on the recursion, however the second type can be derived from a complex structure

Reply:
Fwiw I could fully see the first argument become complex as we improve json support, eg the first argument could be a `SELECT jsonColumn FROM someTable` in future, either way this will catch that quite happily